src/OperatingSystemApp/wobblyLockscreen.js

```diff
@@ -21,7 +21,8 @@ var WobblyLockscreen = GObject.registerClass({
 
     _updateBackground() {
         super._updateBackground();
-        const assetsPath = this.getAssetsPath();
+        // eslint-disable-next-line comma-spacing
+        const [assetsPath,,] = this.getAssetsPath();
```

Comment:
Here you can just do `const [assetsPath] = ...` to get only the first returned element